reporting/report-beacons.go

```diff
@@ -0,0 +1,65 @@
+package reporting
+
+import (
+	"bytes"
+	"html/template"
+	"os"
+
+	"github.com/ocmdev/rita/analysis/beacon"
+	"github.com/ocmdev/rita/database"
+	beaconData "github.com/ocmdev/rita/datatypes/beacon"
+	"github.com/ocmdev/rita/reporting/templates"
+)
+
+func printBeacons(db string, res *database.Resources) error {
+	res.DB.SelectDB(db)
+	var data []beaconData.BeaconAnalysisView
+	ssn := res.DB.Session.Copy()
+	beacon.GetBeaconResultsView(res, ssn, 0).All(&data)
+	ssn.Close()
+
+	return printBeaconHTML(db, data)
+}
+
+func printBeaconHTML(db string, data []beaconData.BeaconAnalysisView) error {
```

Comment:
What do you think of merging the printBeacons functions and packaging them in reporting.html?